addons/escoria-core/game/core-scripts/esc/commands/camera_push.gd

```diff
@@ -4,11 +4,16 @@
 # was following a target (like the player) previously, it will no longer follow
 # this target.
 #
+# Make sure the target is reachable if camera limits have been configured.
+#
 # **Parameters**
 #
-# - *target*: Global ID of the `ESCItem` to push the camera to. If the target
-#   has a child node called `camera_node`, its location will be used. If not,
-#   the location of the target will be used
+# - *target*: Global ID of the `ESCItem` to push the camera to. `ESCItem`s have
+#   a "camera_node" property that can be set to point to a node (usually an
+#   `ESCLocation` node). If the "camera_node" property is empty, `camera_push`
+#	will point the camera at the `ESCItem`s location. If however, the `ESCItem`
```

Comment:
Remove the extra whitespace here and in line 16.

Reply:
Fixed and rebased.